src/main/java/com/hbm/entity/item/EntityDroneBase.java

```diff
@@ -112,7 +112,9 @@ public void onUpdate() {
 					this.motionZ = dist.zCoord * speed;
 				}
 			}
-			
+			if(isCollidedHorizontally){
+				motionY += 1;
```

Comment:
wouldn't this cause drones to shoot up uncontrollably when smacking into the side of a tall wall?

Reply:
its an amazing pathfinding ai

Reply:
Oh yes, but if your drone is running into a gigantic wall, there is something VERY wrong with your setup
In 99% of cases it fixes that stupid bug, and in the 1% it has a harmless side effect